easybuild/easyconfigs/a/annovar/annovar-2016Feb01-foss-2016a-Perl-5.22.1.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Modified by Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'annovar'
+version = '2016Feb01'
```

Comment:
ok, then please include a comment above `version` explaining where you got it from